R/feature.align.R

```diff
@@ -170,6 +170,7 @@ create_aligned_feature_table <- function(all_table,
 #'  when the m/z range is wide. This parameter limits the tolerance in absolute terms. It mostly
 #'  influences feature matching in higher m/z range.
 #' @param do.plot Indicates whether plot should be drawn.
+#' @param sample_names list Given list of sample names.
```

Comment:
The same question as in `compute_clusters.R`.

Reply:
Fixed in [7a08929](https://github.com/RECETOX/recetox-aplcms/pull/153/commits/7a0892911d08febcb2c211dababaa3d98f007abe).